builtin/credential/jwt/path_oidc.go

```diff
@@ -104,6 +109,26 @@ func pathOIDC(b *jwtAuthBackend) []*framework.Path {
 				},
 			},
 		},
+		{
+			Pattern: `oidc/poll`,
+			Fields: map[string]*framework.FieldSchema{
+				"state": {
+					Type: framework.TypeString,
+				},
+				"client_nonce": {
+					Type: framework.TypeString,
+				},
+			},
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.UpdateOperation: &framework.PathOperation{
```

Comment:
Should this be a `logical.ReadOperation`? I don't really view this poll as having updated the state, logically, to a user. Other read operations can update the state from time to time (PKI's CRL read for instance perhaps?), so I don't think we need to expose API operation == storage operation(s).